### PR TITLE
add review

### PR DESCRIPTION
**Pull request by johnnybcs**

### CODE REVIEW

File: client/src/components/AddReview/AddReview.js [hunk 1]
```diff
@@ -16,13 +21,27 @@ import { useState } from "react";
...
   }
 
+  function handleSubmitReview(value) {
+    reviews.push({
+      review_id: reviews.length + 1,
+      user_id: currentUserID,
+      place_id: currentPlaceID,
+      rating: rateValue,
+    });
+    alert(JSON.stringify(reviews));
```
Comment (sassansh) [inline client/src/components/AddReview/AddReview.js]: Great implementation! 👏🏻But let's remove this debugging line. ⚠️

**Review by sassansh — CHANGES_REQUESTED** [review]

**Comment by sassansh** [comment]
image

Something more that can be done on this page is to render the place name, category and image based on the `CurrentPlaceID`

**Comment by johnnybcs** [comment]
@sassansh Thanks for the suggestions. I've made changes so that the place rendered is based on place id. I also added a thank you page that appears after the review is submitted.

**Review by sassansh — APPROVED** [review]
chrome-capture

Tested! Working Beautiful. Go ahead and merge. ✅

**Comment by johnnybcs** [comment]
@sassansh Thanks for the suggestions and code review!

**Pull request merged**